### PR TITLE
Fix secondary chunks when code-splitting

### LINKED ISSUE #177: Possible problem with code-splitting
**_Bug report_**

- [x] I've searched for existing issues
- [x] I've checked documentation: https://docs.frontity.org
- [x] I've checked community forum: https://community.frontity.org

**Describe the expected behavior**

Frontity should load even with code splitting and vendors.

**Describe the observed behavior**

Frontity is not loading in this page:
https://goiblas.com/2019/07/28/como-crear-un-bloque-de-restriccion-de-contenido-para-wordpress/

It looks like there was a resolved issue with this problem:
https://github.com/frontity/frontity/issues/21

Maybe there is a problem with vendors created from a code splitted module.

**Descriped the steps involved to reproduce the problem**

- Go to https://goiblas.com/2019/07/28/como-crear-un-bloque-de-restriccion-de-contenido-para-wordpress/
- Frontity doesn't load

We can take a look at the real project with the code here:
https://github.com/goiblas/personal-blog

### PR DESCRIPTION
**Pull request by luisherranz**
#### Description of what you did:

Secondary chunks (`chunk-1~chunk-2.xxx.js`) didn't work. This PR fixes that.

#### My PR is a:

- 🐞 Bug fix

#### Is the PR ready to be merged?

- ✅ Yes!

Fixes https://github.com/frontity/frontity/issues/177

### CODE REVIEW

File: e2e/packages/loadable/src/dynamic.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const Dynamic: React.FC = () => {
+  const [toggle, setToggle] = React.useState(false);
+  return (
+    <>
+      <div data-test-id="dynamic-div">I am the Dynamic1 component</div>
+      <button data-test-id="toggle-button" onClick={() => setToggle(!toggle)}>
+        Toggle
+      </button>
+      <div data-test-id="toggle-div">{toggle ? "ON" : "OFF"}</div>
+      <Dynamic2 />
```
Comment (DAreRodz) [inline e2e/packages/loadable/src/dynamic.tsx]: I guess there is a reason for adding `<Dynamic2 />` here but why isn't there any e2e test in `loadable.spec.js` looking for that component?

**Review by DAreRodz — APPROVED** [review]
It looks great! Just a couple of questions.

File: e2e/packages/loadable/src/static.tsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+const Static: React.FC = () => {
+  return <div data-test-id="dynamic2-div">I am the Static component</div>;
```
[the remark this answers is not in the document]
Comment (luisherranz) [reply e2e/packages/loadable/src/static.tsx]: Fixed!

File: e2e/packages/loadable/src/dynamic.tsx (hunk 1 above)
Comment (luisherranz) [reply e2e/packages/loadable/src/dynamic.tsx]: It's not actually. Fixed and added test for Dynamic2!

**Comment by luisherranz** [comment]
Thanks for the review. Pushing the changes.

**Pull request merged**